include/odp/api/spec/shared_memory.h

```diff
@@ -213,22 +211,23 @@ int odp_shm_capability(odp_shm_capability_t *capa);
  * Reserve a contiguous block of shared memory
  *
  * Reserve a contiguous block of shared memory that fulfills size, alignment
- * and shareability (ODP_SHM_* flags) requirements. In general, a name is
- * optional and does not need to be unique. However, if the block will be
+ * and shareability (ODP_SHM_* flags) requirements. By default (with no flags), the memory
+ * block can be shared between all ODP threads (pthreads) of the calling process, and with
+ * ODP processes that are forked from the calling process _after_ the reserve call.
```

Comment:
Need to check if memory can be shared between process, although addresses cannot be.